docs/general/session.md

```diff
@@ -0,0 +1,22 @@
+# Semantic conventions for session
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+Session is defined as the period of time encompassing all activities performed by the application and the actions
```

Comment:
Who determines when the session starts and ends? Are there common practices of doing so?

Reply:
Session starts when the mobile app is launched or when a web page is loaded. Session typically ends when app is closed or a timeout is reached.

Reply:
sorry if I was unclear - I think the explanation needs to be in the text, including the "who decides" part.

Reply:
Ideally, we would provide a spec for this behavior, however it may be determined by then instrumentation vendor.

Reply:
@yurishkuro would you mind moving this to an issue for future spec work? I think that could help us get to resolution on this PR. 🙏🏻

Reply:
I agree this should probably be part of the spec. Also the part about "This value should be generated by the SDK and not supplied by the client", seems also like a impl detail and should be part of the SDK spec.